scriptworker/test/test_task.py

```diff
@@ -342,39 +365,73 @@ def test_reclaim_task_non_409(context, successful_queue, event_loop):
         )
 
 
+@pytest.mark.parametrize("proc", (None, 1))
 @pytest.mark.asyncio
-async def test_reclaim_task_mock(context, mocker, event_loop):
+async def test_reclaim_task_mock(context, mocker, proc):
+
```

Comment:
It'd be somewhat easier to review the tests if the had an explanation of what they were trying to test. (I've often pointed people [here](https://jml.io/pages/test-docstrings.html)).

Reply:
Something like

> When `queue.reclaim_task` raises an error with status 409, `reclaim_task` returns. If there is a running process, `reclaim_task` tries to kill it before returning.

Reading that docstring, suggests to me that `fake_reclaim` should actually be the function that is raising the exception, rather than monkey-patching `pformat` to do that.